feat(security): add new section (#18779)
newrelic/docs-website: src/content/docs/security/security-privacy/data-privacy/data-privacy-new-relic.mdx

            </th>
            <th>
                Event description
            </th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                [`NRAuditEvent`](/attribute-dictionary/?event=NrAuditEvent)
            </td>
            <td>
                Records user activity when a user makes service configuration changes within your New Relic organization 
            </td>
        </tr>
        <tr>
            <td>
                [`NrdbQuery`](/attribute-dictionary/?event=NrdbQuery)
            </td>
            <td>
                Records user activity when a user queries data within the account
            </td>
        </tr>
    </tbody>
</table>

You can surface user activity data by going to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities**, then working with two capabilities: **Metrics & Events** and **Query your data**. In general, you can use:

* Metrics & Events for looking at broad, general trends
* Query your data for answering specific, scoped questions with NRQL queries 

<CollapserGroup>
    <Collapser
        id="metrics-events"
        title="Surface event data in Metrics & Events"
    >

> To surface user activity information, go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > Metrics & Events**, then search for either `NrAuditEvent` or `NrdbQuery` from the **Event type** column. You can click the **Dimensions** dropdown to view all attributes associated with either event. For example:

Author: ```suggestion
To surface user activity information, go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > Metrics & Events**, then search for either `NrAuditEvent` or `NrdbQuery` from the **Event type** column. For example:
```

<img
    title="Find user activity data in Metrics & Events"

>     alt="A screenshow that shows how to find events and attributes for user activity data"

Author: ```suggestion
    alt="A screenshot that shows how to find events and attributes for user activity data"
```

    src={nr1FindUserActivityDatainMetricsandEvents}
/>

<figcaption>
    Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > Metrics & Events**: Search `NrAuditEvent` or `NrdbQuery` from the **Event type** column, then click **Dimensions** to view attributes.
</figcaption>

> You can select **Raw data** to view event attributes as key-value pairs. Attribute keys like ID, description, timestamp are organized in the table's horizontal row while their value pairs are listed out in the table's columns. These attributes are subject to [data limits](/docs/data-apis/manage-data/view-system-limits). 

Author: ```suggestion
You can select **Raw data** to view event attributes as key-value pairs. Attribute keys like ID, description, or timestamp are organized in the table's horizontal row while their values are listed out in the table's columns. Keep in mind that these attributes are subject to [data limits](/docs/data-apis/manage-data/view-system-limits). 
```

    </Collapser>
    <Collapser
        id="query-your-data"
        title="Query event data with NRQL"
    >

> You can also use our **Query your data** tool, anchored at the bottom of the platform in your browser, to retrieve attributes and fields of interest. For example, take this example query:

Author: ```suggestion
You can also use our **Query your data** tool to retrieve attributes and fields of interest. For example, take this example query:
```

    ```sql
    FROM NrdbQuery SELECT user, query, productCapability, source.name where user = 'demonewrelic@gmail.com' limit max since 1 week ago until 1 day ago
    ```

<img
    title="Query user activity data"

>     alt="A screenshow that shows how to find events and attributes for user activity data"

Author: ```suggestion
    alt="A screenshot that shows how to query events to answer specific questions about user activity data"
```

    src={nr1FindUserActivityDatainNrql}
/>

<figcaption>
    Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > Query your data**: Build a query to answer specific questions about user activity.
</figcaption>

This query surfaces data from the `NrdbQuery` event but limits the data to:

     * These attributes: `SELECT user, query, productCapability, source.name`
     * A specific user: `where user = 'demonewrelic@gmail.com'`
     * With no set maximum number of results: `limit max`

>      * But scoped to this time parameter: `since 1 week ago until 1 day ago`

Author: ```suggestion
     * Scoped to this time parameter: `since 1 week ago until 1 day ago`
```

> This surfaces data to specific parameters rather than showing broad trends about the `NrdbQuery` event like in Metrics & Events. 

Author: ```suggestion
This surfaces data about specific parameters rather than showing broad trends like in Metrics & Events. 
```
